tests/integration/targets/provision_vm_test/vars.yml

```diff
@@ -0,0 +1,12 @@
+mock_file: "provision_vm_test"
+
+provision_vm_hostname: "127.0.0.1:1080"
```

Comment:
I think you want to either remove this file or add the variables from `tasks/main.yml` here.

Reply:
Removed it